spec/lita/handlers/totems_spec.rb

```diff
@@ -84,6 +84,7 @@ def send_destroy_message
     context "totem exists" do
       before do
         send_message("totems create chicken")
+        send_message("totems create noir")
```

Comment:
you add it but seems you never use the noir totem?

Reply:
oh, yeah, thanks! I intended to use it to test the timeout